src/systemd.cr

```diff
@@ -21,8 +21,26 @@ module SystemD
     self.notify("STATUS=#{status}\n")
   end
 
-  def self.watchdog
-    self.notify("WATCHDOG=1\n")
+  def self.start_watchdog
```

Comment:
I think the change makes sense but change the name of the method will break any usage, why not just keep name as `watchdog`?